projects/swimlane/ngx-ui/src/lib/components/calendar/calendar.component.scss

```
@@ -1,22 +1,25 @@
 @import "colors/index";
 
 $calendar-bg: $color-blue-grey-800;
-$calender-border: $color-blue-grey-700;
+$calendar-border: $color-blue-grey-700;
 $calendar-title-bg: $color-blue-grey-800;
 $calendar-title-btn: $color-blue-grey-350;
 $calendar-hover-bg:  $color-blue-400;
 $calendar-today-bg: $color-blue-grey-750;
 $calendar-active-bg:  $color-blue-400;
 $calendar-active-text: $color-grey-050;
 $calendar-day-text: $color-blue-grey-350;
+$calendar-month-current : #262C38;
```

Let's try avoiding hardcoding colors and use the color variables instead. You can find them all [here].(https://swimlane.github.io/ngx-ui/#colors)
The value here can be replaced with `$color-blue-grey-750`

```suggestion
$calendar-month-current : $color-blue-grey-750;
```